files/zh-cn/mozilla/firefox/experimental_features/index.md

```diff
@@ -1,656 +1,2713 @@
-| **{{HTMLElement("dialog")}}** 包含与之交互的 DOM 元素的对话框元素。Implementation of modal dialogs and accessibility still missing.                                            | 关闭 <sub>(从 53 开始支持)</sub>                     | 关闭 <sub>(从 53 开始支持)</sub> | ---                              | ---            | `dom.dialog_element.enabled` |
+### input type="search" 外观
+
+输入类型 `type="search"` 的布局已更新。一旦有人开始输入，搜索字段就会有一个清晰的图标，以便与其他浏览器的实现相匹配。（详情请参见 [Firefox bug 558594](https://bugzil.la/558594)）。
+
+<table>
+  <thead>
+    <tr>
+      <th>发布渠道</th>
+      <th>添加的版本</th>
+      <th>默认启用？</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>Nightly</th>
+      <td>81</td>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th>开发者版</th>
+      <td>81</td>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th>Beta</th>
+      <td>81</td>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th>正式发布版</th>
+      <td>81</td>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th>首选项名称</th>
+      <td colspan="2"><code>layout.forms.input-type-search.enabled</code></td>
+    </tr>
+  </tbody>
+</table>
+
+### 切换密码显示
+
+HTML 密码输入元素（[`<input type="password">`](/zh-CN/docs/Web/HTML/Element/input/password)）包含一个“眼睛”图标，可通过切换来显示或遮盖密码文本（[Firefox bug 502258](https://bugzil.la/502258)）。
+
+<table>
+  <thead>
+    <tr>
+      <th>发布渠道</th>
+      <th>添加的版本</th>
+      <th>默认启用？</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>Nightly</th>
+      <td>96</td>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th>开发者版</th>
+      <td>96</td>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th>Beta</th>
+      <td>96</td>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th>正式发布版</th>
+      <td>96</td>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th>首选项名称</th>
+      <td colspan="2"><code>layout.forms.input-type-show-password-button.enabled</code></td>
+    </tr>
+  </tbody>
+</table>
+
+### ARIA 属性反射
+
+为非 IDREF 属性启用了 [ARIA](/zh-CN/docs/Web/Accessibility/ARIA) 反射功能，允许作者通过 JavaScript API 直接获取和设置 DOM 元素上的 ARIA 属性，而不是使用 `setAttribute` 和 `getAttribute`（详情请参见 [Firefox bug 1824980](https://bugzil.la/1824980)）。
+
+<table>
+  <thead>
+    <tr>
+      <th>发布渠道</th>
+      <th>添加的版本</th>
+      <th>默认启用？</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>Nightly</th>
+      <td>114</td>
+      <td>是</td>
+    </tr>
+    <tr>
+      <th>开发者版</th>
+      <td>114</td>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th>Beta</th>
+      <td>114</td>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th>正式发布版</th>
+      <td>114</td>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th>首选项名称</th>
+      <td colspan="2"><code>accessibility.ARIAReflection.enabled</code></td>
+    </tr>
+  </tbody>
+</table>
 
 ## CSS
 
+### 显示杂散控制字符的十六进制框
+
+该特性会将除 _tab_（`U+0009`）、_line feed_（`U+000A`）、_form feed_（`U+000C`）和 _carriage return_（`U+000D`）之外的控制字符（Unicode 类别 Cc）渲染为十六进制框，而这些控制字符并非预期字符。（详见 [Firefox bug 1099557](https://bugzil.la/1099557)）。
```

Comment:
could we translate the key name:

```suggestion
该特性会将除*制表符*（`U+0009`）、_换行符_（`U+000A`）、_换页符_（`U+000C`）和*回车符*（`U+000D`）以外的控制字符（Unicode 类别 Cc）渲染为十六进制框，而这些控制字符并非预期字符。（详见 [Firefox bug 1099557](https://bugzil.la/1099557)）。
```

I'm not sure about the translation of "hexbox".

Reply:
Some old bug report document have some screenshots, the malfunctioned UI text look like 'hexbox'es 👀 

![image](https://github.com/mdn/translated-content/assets/40999116/6778694f-ebe5-4878-b1f7-df66dec4032b)

Reply:
Thanks for the clarification @jasonren0403 :+1: